src/agent/io/sourcemapper.ts

```diff
@@ -167,17 +174,10 @@ async function processSourcemap(
 }
 
 export class SourceMapper {
+  /** Maps each original source path to the corresponding source map info. */
   infoMap: Map<string, MapInfoInput>;
 
-  /**
-   * @param {Array.<string>} sourcemapPaths An array of paths to .map sourcemap
-   *  files that should be processed.  The paths should be relative to the
-   *  current process's current working directory
-   * @param {Logger} logger A logger that reports errors that occurred while
-   *  processing the given sourcemap files
-   * @constructor
-   */
-  constructor() {
+  constructor(readonly logger: Logger) {
```

Comment:
Private constructor?

Reply:
The create() function is actually not a static function of this class, marking the constructor as private will change the usage of this function in all places. I think for this PR, it is better to keep it as it is, what do you think?